### PR TITLE
Add dotnet foundation youtube channel button

### PR DESCRIPTION
**Pull request by sejka**
due to https://youtu.be/7aXynXtrDco?t=249
made live on hacktoberfest poznań (https://hacktoberfest.pl/)

### CODE REVIEW

**Comment by hishamco** [comment]
Could you please provide a screenshot of how it looks like?

**Comment by sejka** [comment]
image
image
Thanks!

**Comment by hishamco** [comment]
LGTM, waiting for @DamianEdwards approval

**Comment by sejka** [comment]
@jongalloway Maybe we could get nicer channel id?

**Comment by sejka** [comment]
I moved channel id to app settings

File: src/live.asp.net/ViewModels/HomeViewModel.cs [hunk 1]
```diff
@@ -38,6 +38,10 @@ public class HomeViewModel
 
         public bool ShowMoreShowsUrl => !string.IsNullOrEmpty(MoreShowsUrl);
 
+        public string YoutubeChannelId { get; set; }
```
Comment (hishamco) [inline src/live.asp.net/ViewModels/HomeViewModel.cs]: YouTube is better than Youtube 😄

File: src/live.asp.net/ViewModels/HomeViewModel.cs [hunk 2]
```diff
@@ -38,6 +38,10 @@ public class HomeViewModel
...
+
+        public string YoutubeSubscribeLink => $"https://www.youtube.com/channel/{YoutubeChannelId}?sub_confirmation=1";
```
Comment (hishamco) [inline src/live.asp.net/ViewModels/HomeViewModel.cs]: Use YouTubeSubscriptionLink

**Review by hishamco — CHANGES_REQUESTED** [review]

**Comment by hishamco** [comment]
Please squash your commits

**Pull request closed without merging**